kafka-connect-s3/pom.xml

```diff
@@ -197,6 +197,53 @@
                             </dependency>
                         </dependencies>
                     </plugin>
+                    <plugin>
+                        <groupId>io.confluent</groupId>
+                        <version>0.9.0</version>
```

Comment:
This version generates incorrectly-structured archive files, so we'll need to make sure to update this to either `0.9.1` or `1.0.0` (whichever is available at the time) before merging this PR.

Reply:
Fixed other comments. Will update this once you push your changes. Thanks!